DQM/SiStripMonitorCluster/src/SiStripMonitorCluster.cc

```diff
@@ -926,6 +982,17 @@ void SiStripMonitorCluster::analyze(const edm::Event& iEvent, const edm::EventSe
 	    }
 
 	  }
+
+    //Filling # clusters per FED ID histogram from FED Id clusters map (for all layers simultaneously).
+    map<int,int>::iterator it;
+    for(it=FEDID_v_clustersum.begin(); it!=FEDID_v_clustersum.end(); it++){
```

Comment:
please use a range based loop here.
for ( auto const & fedidEntry : FEDID_v_clustersum ) NumberOfFEDClus->Fill(fedidEntry.first......